doc/opt_aero.rst

```diff
@@ -93,20 +93,42 @@ We can set up constraints on the geometry with the DVConstraints class, also fou
 There are several built-in constraint functions within the DVConstraints class, including thickness, surface area, volume, location, and general linear constraints.
 The majority of the constraints are defined based on a triangulated surface representation of the wing obtained from ADflow.
```

Comment:
Can you add a few more lines here stating that the triangulated surface is created by ADflow using the wall surfaces defined on the CFD volume mesh. The resolution is on a similar level of the CFD surface mesh, and users do not need to provide a triangulation themselves. Optionally, I think the surface can also be defined with an external file, see the details here: https://github.com/mdolab/pygeo/blob/master/pygeo/DVConstraints.py#L200
I think this would be useful if users wanted to have a different resolution on the triangulated surface (finer or coarser) compared to the CFD mesh, or if DVConstraint was being used without ADflow.

Reply:
Added a note.